src/test/java/io/swagger/oas/test/integration/responses/PrimitiveResponseTestIT.java

```diff
@@ -152,10 +152,54 @@ public void verifyGetEmailResponse() throws Exception {
     public void verifyGetMapResponse() throws Exception {
         Map<String, String> queryParams = new HashMap<String, String>();
 
-        String str = client.invokeAPI("/mockResponses/primitiaveMapResponse", "GET", queryParams, null, new HashMap<String, String>(), null, "application/json", null, new String[0]);
+        String str = client.invokeAPI("/mockResponses/primitiveMapResponse", "GET", queryParams, null, new HashMap<String, String>(), null, "application/json", null, new String[0]);
         assertEquals(str, "{\"key\":\"key\",\"value\":\"string\"}");
     }
 
+    /**
+     * verifies that the email return value is automatically created
+     */
+    @org.junit.Test
+    public void verifyAdditionalPropertyResponse() throws Exception {
+        Map<String, String> queryParams = new HashMap<String, String>();
+
+        String str = client.invokeAPI("/mockResponses/additionalPropertiesTest", "GET", queryParams, null, new HashMap<String, String>(), null, "application/json", null, new String[0]);
+        assertEquals(str, "{\"key\":\"key\",\"value\":\"string\"}");
+    }
+
+    /**
+     * verifies that the email return value is automatically created
+     */
+    @org.junit.Test
+    public void verifyAdditionalPropertyIntegerResponse() throws Exception {
+        Map<String, String> queryParams = new HashMap<String, String>();
+
+        String str = client.invokeAPI("/mockResponses/additionalProperties1", "GET", queryParams, null, new HashMap<String, String>(), null, "application/json", null, new String[0]);
+        assertEquals(str, "{\"key\":\"key\",\"value\":{\"bar\":0}}");
+    }
```

Comment:
doesn't seem right, should be something like `"{\"foo\":0,\"additionalPropertyExampleKey\":{\"bar\":0}}}"`

Reply:
I changed the code, and try not to break the older tests in the process